mesonbuild/modules/gnome.py

```diff
@@ -886,6 +885,107 @@ def mkenums(self, state, args, kwargs):
         else:
             return ModuleReturnValue(targets, targets)
 
+    def mkenums_simple(self, state, args, kwargs):
+        hdr_filename = args[0] + '.h'
+        body_filename = args[0] + '.c'
+
+        # not really needed, just for sanity checking
+        forbidden_kwargs = ['c_template', 'h_template', 'eprod', 'fhead',
```

Comment:
Actually, what I wrote makes no sense because I only copy over specific fields via a whitelist already, so there can be no leakage. Just unused args in meson.build file.